exercises/practice/pov/src/lib.rs

```diff
@@ -0,0 +1,34 @@
+use std::fmt::Debug;
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct Tree<T: Clone + Debug + PartialEq> {
+    _remove_this: std::marker::PhantomData<T>,
+}
+
+impl<T: Clone + Debug + PartialEq> Tree<T> {
+    pub fn new(label: T) -> Self {
+        todo!("Implement a function that creates a new Tree with given {label:?}");
+    }
+
+    pub fn with_children(label: T, children: Vec<Self>) -> Self {
+        todo!("Implement a function that creates a new Tree with given {label:?} and {children:?}");
+    }
+
+    pub fn get_label(&self) -> T {
+        todo!("Implement getter for label.");
+    }
+
+    pub fn get_children(&self) -> Vec<&Self> {
```

Comment:
The [Rust API guidelines](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter) discourage a `get_` prefix for getter names. Rename these to just `label` and `children`.